### ISSUE TITLE
[unconfirmed bug] wrong perm req when autostart / autostart toggle requested

### ISSUE THREAD

**Issue by nonlin-lin-chaos-order-etc-etal** [PurpleI2P/i2pd-android#63]
```
[22:30:37] conscius> можно i2p-android будет когда-нибудь потыкать
[22:31:17] conscius> я ему говорю запускайся автостартом, а он мне 
дает список приложений которые умеют себя рисовать поверх других приложений
[22:31:22] conscius> не ту пермиссию запрашивает
```

**Comment by nonlin-lin-chaos-order-etc-etal** [comment]
Fixed at commit 55986458f2ebdc77350e53e22323c8c5e677c095

**Comment by nonlin-lin-chaos-order-etc-etal** [comment]
https://github.com/PurpleI2P/i2pd-android/commit/55986458f2ebdc77350e53e22323c8c5e677c095#diff-25f93956de3707de5cce0f344e9e6f22b35ac9fed56b1564d67d0cbd9b8bb518L75
